package/samplers/llambo/README.md

```diff
@@ -0,0 +1,411 @@
+---
+author: [Jinglue Xu]
```

Comment:
[@tennison](https://github.com/tennisonliu)
Would you like to be one of the authors as this method is from your paper?

Reply:
Same here, thanks!